src/main/webapp/components/index-cards/index-cards.js

```diff
@@ -108,13 +108,13 @@ export const IndexCardItem = props => {
           subheader={subHeader || moment(modified).fromNow()}
           action={headerAction}
         />
-        <CardContent>
-          {metacard_owner && (
+        {metacard_owner && (
```

Comment:
Helps shrink the space between the title and the actions when there's no metacard owner.